virtool/flags.py

```diff
@@ -1,46 +1,50 @@
-from typing import List, Callable, Dict
+from typing import List, Callable
 from enum import Enum
 from aiohttp import web
 from aiohttp.web_exceptions import HTTPNotFound
 from aiohttp.web import Request
 
-import virtool.http.routes
-from virtool.types import App
-
-routes = virtool.http.routes.Routes()
-
 
 class FlagName(Enum):
-    FF_SPACES = "FF_SPACES"
-    FF_ML_MODELS = "FF_ML_MODELS"
-    FF_ADMINISTRATOR_ROLES = "FF_ADMINISTRATOR_ROLES "
+    SPACES = "SPACES"
+    ML_MODELS = "ML_MODELS"
+    ADMINISTRATOR_ROLES = "ADMINISTRATOR_ROLES"
 
 
 class FeatureFlags:
     """
-    Class contains the derived value of the feature flags and cli flag config.
+    Class contains the derived value of the feature flags.
     """
 
-    def __init__(self):
+    def __init__(self, cli_flags: List[FlagName]):
```

Comment:
The `FeatureFlags` class doesn't care where the flags are passed from (good design). Using `cli_` implies coupling unnecessarily.

```suggestion
    def __init__(self, overrides: List[FlagName]):
```